### PR TITLE
Feature/abx 1234 expose required pdfium functionality

### PR DESCRIPTION
**Pull request by danielklokov-airbox**
Ticket ABX-1234

- project dependencies upgraded to AndroidX
- exposed methods required to get document dimensions and page offset relative to the document
- imported PdfiumAndroid module directly into project
- updated sample app to demo exposed functionality

### CODE REVIEW

File: android-pdf-viewer/src/main/java/com/github/barteksc/pdfviewer/PDFView.java [hunk 1]
```diff
@@ -1105,6 +1105,65 @@ public float toCurrentScale(float size) {
...
+     * Width is the page with maximum width and height is total height of all pages.
+     * @return document size
+     */
+    public SizeF getDocumentSize() {
+        return  getDocumentSize(getZoom());
+    }
+
+    /**
+     * Get document size taking into account specific scale (zoom).
+     * Width is the page with maximum width and height is total height of all pages.
```
Comment (hwd71) [inline android-pdf-viewer/src/main/java/com/github/barteksc/pdfviewer/PDFView.java]: typo - with -> width

File: android-pdf-viewer/src/main/java/com/github/barteksc/pdfviewer/PDFView.java [hunk 2]
```diff
@@ -1105,6 +1105,65 @@ public float toCurrentScale(float size) {
...
+     */
+    public SizeF getDocumentSize(float zoom) {
+        if (pdfFile == null) {
+            return null;
+        }
+
+        float documentWidth = pdfFile.getMaxPageSize().getWidth() * zoom;
+        float documentHeight = pdfFile.getDocLen(zoom);
+        SizeF documentSize = new SizeF(documentWidth, documentHeight);
+        return  documentSize;
```
Comment (hwd71) [inline android-pdf-viewer/src/main/java/com/github/barteksc/pdfviewer/PDFView.java]: double space?

File: sample/src/main/java/com/github/barteksc/sample/PDFViewActivity.java [hunk 3]
```diff
@@ -223,4 +252,173 @@ public void onRequestPermissionsResult(int requestCode, @NonNull String permissi
...
+            canvas.drawPath(mAnnotationDrawingPath, paint1);
+        }
+
+        // Draw cached annotations
+        Paint paint2 = new Paint();
+        paint2.setStyle(Paint.Style.STROKE);
+        paint2.setStrokeWidth(10);
+        paint2.setColor(Color.RED);
+
+        Log.d("OnDrawLayer", "############ ANNOTATIONS #################");
```
Comment (hwd71) [inline sample/src/main/java/com/github/barteksc/sample/PDFViewActivity.java:273]: that's not spamming the device log on every draw is it? I guess .d won't do this on the actual device?

**Review by hwd71 — APPROVED** [review]
Lovely.

File: sample/src/main/java/com/github/barteksc/sample/PDFViewActivity.java (hunk 3 above)
Comment (danielklokov-airbox) [reply sample/src/main/java/com/github/barteksc/sample/PDFViewActivity.java:273]: This is just sample app to test if all changes are working as expected

File: android-pdf-viewer/src/main/java/com/github/barteksc/pdfviewer/PDFView.java (hunk 1 above)
Comment (danielklokov-airbox) [reply android-pdf-viewer/src/main/java/com/github/barteksc/pdfviewer/PDFView.java]: What I was trying (badly) to say that document width is equal to the largest page width and height is total of all page heights.

**Review by hwd71 — APPROVED** [review]

**Pull request merged**